reactor-core/src/main/java/reactor/core/publisher/Hooks.java

```diff
@@ -525,9 +606,18 @@ public static void resetOnNextError() {
 	static final Map<String, BiFunction<? super Throwable, Object, ? extends Throwable>> getOnOperatorErrorHooks() {
 		return Collections.unmodifiableMap(onOperatorErrorHooks);
 	}
+	static final Map<String, Function<Runnable, Runnable>> getOnScheduleHooks() {
+		return Collections.unmodifiableMap(onScheduleHooks);
+	}
 
 	static final Logger log = Loggers.getLogger(Hooks.class);
 
+	/**
+	 * This key is used to add the {@link #onSchedule(String, Function)} composite hook as
+	 * a {@link Schedulers#addExecutorServiceDecorator(String, BiFunction) executorService decorator}.
+	 */
+	private static final String EXECUTOR_DECORATOR_ONSCHEDULE = "reactor.onSchedule";
```

Comment:
@bsideup I changed the constant name a bit and made it private because other constants at this level are used for storing a hook override in the `Context` (hence the `.local` suffix). This is different though: a key for the `Schedulers` own "hook" (the executor decorator).